src/util.js

```diff
@@ -306,14 +306,15 @@ define([
         }
 
         var form = mug.form,
+            vellumKey = key.replace(':', '__'),
```

Comment:
Would be up for changing this. Not that concerned since we do control the vellum namespace, but probably not the most ideal situation